examples/sgd/sgd.go

```diff
@@ -20,4 +20,6 @@ func main() {
 		opt.Step()
 	}
 	torch.FinishGC()
+	opt.Close()
```

Comment:
Can we use `FinishGC()` to close the `opt` and `a`?

Reply:
`GC` and `FinishGC` are a pair of APIs.  The variables defined between `GC` and `FinishGC` will be released.

`opt` and `a` are defined out of the scope, so we have to close them manually.